rllib/algorithms/algorithm_config.py

```diff
@@ -332,15 +332,9 @@ def __init__(self, algo_class=None):
 
         # `self.explore()`
         self.explore = True
-        self.exploration_config = {
-            # The Exploration class to use. In the simplest case, this is the name
-            # (str) of any class present in the `rllib.utils.exploration` package.
-            # You can also provide the python class directly or the full location
-            # of your class (e.g. "ray.rllib.utils.exploration.epsilon_greedy.
-            # EpsilonGreedy").
-            "type": "StochasticSampling",
-            # Add constructor kwargs here (if any).
-        }
+        # This is not compatible with RLModules, which have a method
+        # `forward_exploration` to specify custom exploration behavior.
+        self.exploration_config = {}
```

Comment:
@sven1977 The idea here is that AlgorithmConfig does not include exploration config so that we don't end up with that being the default everywhere anymore.
This is important because we sometimes merge configs with AlgorithmConfig to get sensible defaults for values.

Reply:
Makes sense. I approved. Waiting for @zhe-thoughts for approval so we can merge after the master freeze.